doc/Getting-started/index.md

```diff
@@ -65,543 +41,211 @@ To use Kaa Sandbox, your system must meet the following minimum system requireme
-                }
-            });
-
-            // Starts Kaa SDK client
-            kaaClient.start();
-        }
-    }
-```
+You can access Admin UI by clicking **Administrative console** at the top of the window. Refer to the [Admin UI guide]({{root_url}}Administration-guide/Tenants-and-applications-management) for working instructions.
 
-</div><div id="Cpp" class="tab-pane fade" markdown="1">
-
-```C
-    #include <cstdint>
-    #include <iostream>
-    #include <memory>
-
-    #include <kaa/Kaa.hpp>
-    #include <kaa/notification/INotificationListener.hpp>
-    #include <kaa/notification/INotificationTopicListListener.hpp>
-
-    using namespace kaa;
-
-    class SimpleKaaClientStateListener : public IKaaClientStateListener {
-    public:
-        virtual void onStarted() {
-            std::cout << "Kaa SDK client started!" << std::endl;
-        }
-
-        virtual void onStartFailure(const KaaException& exception) {}
-        virtual void onPaused()  {}
-        virtual void onPauseFailure(const KaaException& exception) {}
-        virtual void onResumed() {}
-        virtual void onResumeFailure(const KaaException& exception) {}
-
-        virtual void onStopped() {}
-        virtual void onStopFailure(const KaaException& exception) {}
-    };
-
-    class BasicNotificationTopicListListener : public INotificationTopicListListener {
-    public:
-        virtual void onListUpdated(const Topics& topics)
-        {
-            std::cout << "Topic list was updated" << std::endl;
-            for (const auto& topic : topics) {
-            std::cout << "Received topic with id " << topic.id << " and name '" << topic.name << "'" << std::endl;
-        }
-        }
-    };
-
-    class BasicNotificationListener : public INotificationListener {
-    public:
-        virtual void onNotification(const std::int64_t topicId, const KaaNotification& notification)
-        {
-            std::cout << "Received notification '" << notification.message << "'"
-                      << "for topic with id '" << topicId  << "'" << std::endl;
-        }
-    };
-
-    int main()
-    {
-        BasicNotificationTopicListListener topicListListener;
-        BasicNotificationListener commonNotificationListener;
-        // Create client for Kaa SDK
-        auto kaaClient = Kaa::newClient(std::make_shared<KaaClientPlatformContext>(),
-                                    std::make_shared<SimpleKaaClientStateListener>());
-        kaaClient->addTopicListListener(topicListListener);
-        kaaClient->addNotificationListener(commonNotificationListener);
-        // Start Kaa SDK client
-        kaaClient->start();
-
-        std::cout << "Presss any key to stop Kaa SDK client" << std::endl;
-        std::getchar();
-
-        // Stop Kaa SDK client
-        kaaClient->stop();
-
-        return 0;
-    }
-```
+> **NOTE:** Kaa Sandbox provides default credentials for all three types of Kaa users, as follows:
+>
+> * Kaa admin - kaa/kaa123
```

Comment:
Please, add a clarification that the pair is login/password. It would be also good to put them in backticks (i.e., verbatim).